integration-test/codecept.json

```diff
@@ -0,0 +1,34 @@
+{
+  "output": "./output",
+  "helpers": {
+    "Puppeteer": {
+      "url": "http://localhost",
+      "show": false,
+      "waitForNavigation": "networkidle0",
+      "waitForAction": 500,
+      "chrome": {
+        "args": [
+          "--no-sandbox",
+          "--disable-setuid-sandbox"
+        ]
+      }
+    }
+  },
+  "include": {
+    "I": "./support/steps_file.js",
```

Comment:
Can we rename `I` to something meaningful?

Reply:
`I` is codecept default. I would rather keep that, so that somebody new to the game just has to read the codecept docs and finds everything as expected.

Reply:
But in `TodoList` you are also using `I`. What I struggle with is `steps_file` does not tell me anything (should be renamed to kebab-case here as well) and `I` neither. Is it maybe just a test setup and reused in other tests (then it could be named "test-setup" for example)?